tests/src/Cli/OptimizeTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace PageExperience\Tests\Cli;
+
+use AmpProject\Cli\Command;
+use AmpProject\Cli\Executable;
+use AmpProject\Cli\Options;
+use org\bovigo\vfs\vfsStream;
+use PageExperience\Cli\Command\Optimize;
+use PageExperience\Tests\TestCase;
+use AmpProject\Exception\Cli\InvalidArgument;
```

Comment:
Imports should always be sorted.
```suggestion
use AmpProject\Cli\Command;
use AmpProject\Cli\Executable;
use AmpProject\Cli\Options;
use AmpProject\Exception\Cli\InvalidArgument;
use org\bovigo\vfs\vfsStream;
use PageExperience\Cli\Command\Optimize;
use PageExperience\Tests\TestCase;
```